api/utils.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from pathlib import Path
+
+from werkzeug.datastructures.file_storage import FileStorage
+ALLOWED_EXTENSIONS = {'pptx'}
+
+
+def allowed_file(filename: str) -> bool:
+    """
+    Returns whether the file is within allowed types.
+    :param filename: str file name.
+    :return:
+    """
+    return '.' in filename and \
```

Comment:
It could've been better to separate this code into different lines in order to make in more understandable.